Userland/Libraries/LibJS/AST.h

```diff
@@ -636,10 +636,10 @@ class FunctionNode {
     Vector<FunctionParameter> const m_parameters;
     const i32 m_function_length;
     FunctionKind m_kind;
-    bool m_is_strict_mode { false };
-    bool m_might_need_arguments_object { false };
-    bool m_contains_direct_call_to_eval { false };
-    bool m_is_arrow_function { false };
+    bool m_is_strict_mode : 1 { false };
```

Comment:
Commit `LibJS: Shrink FunctionNode by using bitfields` says it shrinks the class by "a total of bytes" :^)

Reply:
Frick 🤦🏻